src/modules/picklist/__tests__/processBarcode-test.js

```diff
@@ -108,7 +108,7 @@ describe('processBarcode', () => {
   });
 
   it('should update move dialog when receiving a number that resolves to a single object when move dialog is active', () => {
```

Comment:
Name on this test should perhaps be changed, since you now tests that an error should occur?